### PR TITLE
add tutorial

### PR DESCRIPTION
**Pull request by SeaOfOcean**

### CODE REVIEW

File: contents/handbook-tutorials.html [hunk 1]
```diff
@@ -1 +1,149 @@
...
+		<strong>Please bring your laptop to tutorials</strong>. You often need it to show your learning activity results. At other
+		times, we ask you to work on project related things with your team members, which too may require the laptop.
+	</li>
+	<li>
+		<strong>Pay attention to discussions</strong>: When the tutor is discussing something with one team member (e.g., one team
+		member is explaining how he/she used assertions), <strong>the whole team is expected to pay attention</strong> to that
+		discussion, not do your own things. Whatever discussed is relevant to all of you, not just that team member.
+	</li>
+	<strong>Cf</strong>: <a href="https://docs.google.com/document/d/1FhJVBIsip6ShOdZ81bmssllDi
+	z04YBBqoxpZv26_iQE/pub?embedded=true#h.u747bava6j91">What if I don't carry a laptop around?</a>
```
Comment (damithc) [inline contents/handbook-tutorials.html]: Shouldn't this be `class="dim"`?

File: contents/handbook-tutorials.html [hunk 2]
```diff
@@ -1 +1,149 @@
...
+		<li>
+			This is how I understood/applied dependency injection. Is that correct? - Such questions are welcome.
+		</li>
+		<strong>Reason</strong>:This question shows you have put in some effort to learn the topic and seeking
+		further clarification from the tutor.
+		<br>
+		<li>
+			What is dependency injection? - Such questions are not allowed.
+		</li>
+		<strong>Reason</strong>: This question implies you haven’t done what you could to learn the topic in
```
Comment (damithc) [inline contents/handbook-tutorials.html]: Some lines are too long.

**Comment by damithc** [comment]
Some minor things to fix.

**Comment by SeaOfOcean** [comment]
updated, thanks for suggestions.

**Pull request merged**